indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -179,3 +179,15 @@ def _process(self):
             return jsonify_data(page_html=render_abstract_page(self.abstract, management=self.management))
         tpl = get_template_module('events/abstracts/abstract/review.html')
         return jsonify_data(box_html=tpl.render_review_box(form, self.abstract, self.track, management=self.management))
+
+
+class RHEditReviewedForTrackList(RHAbstractReviewBase):
+    CSRF_ENABLED = True
+
+    def _process(self):
+        form = AbstractReviewedForTracksForm(event=self.event_new)
+        if form.validate_on_submit:
+            form.populate_obj(self.abstract)
+            return jsonify_data(page_html=render_abstract_page(self.abstract, management=self.management))
+        tpl = get_template_module('events/abstracts/abstract/review.html')
+        return jsonify_data(box_html=tpl.render_reviewed_for_tracks_box(self.abstract, form=form, management=self.management))
```

Comment:
I think that after we merge everything we should try to look for a more elegant solution for this... it's a pattern that is used in at least 2 other places (same on client side).